core/app/models/spree/address.rb

```diff
@@ -28,6 +28,12 @@ class Address < Spree::Base
 
     self.allowed_ransackable_attributes = %w[name]
 
+    enum :reverse_charge_status, {
+      disabled: 0,
+      enabled: 1,
+      not_validated: 2
+    }
```

Comment:
Can we need prefix this enum, please?

```suggestion
      , prefix: true
    }
```

Reply:
Resolved!